recipes/libtensorflow/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set version = "1.13.1" %}
+
+package:
+  name: libtensorflow-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/tensorflow/archive/v{{ version }}.tar.gz  # [linux]
+  sha256: 7cd19978e6bc7edc2c847bce19f95515a742b34ea5e28e4389dade35348f58ed  # [linux]
+
+  patches:
+    - patches/0001-add-rt-to-linkopts.patch  # [linux]
```

Comment:
Only this first patch is needed since the Python tests are not included nor run during the build.